prerendering.bs

```diff
@@ -658,6 +673,37 @@ The <dfn attribute for="PerformanceNavigationTiming">activationStart</dfn> gette
 
 1. Return a {{DOMHighResTimeStamp}} with a time value equal to the [=current document=]'s [=Document/activation start time=].
 
+<h3 id="client hint cache">Interaction with Client Hint Cache</h3>
+
+Each [=prerendering browsing context=] maintains its <dfn for="prerendering browsing context">prerender scoped Accept-CH cache</dfn>, to persist the origin's opt-in to receive the indicated header fields.
+
+[=prerendering browsing context/Prerender scoped Accept-CH cache=] is an [=ordered map=] of [=origin=] to [=client hints set=].
+
+<div algorithm="add a new prerender Accept-CH cache entry">
+
+Similar to [=add a new Accept-CH cache entry=], to <dfn>add a new prerender Accept-CH cache entry</dfn> to the [=prerendering browsing context/prerender scoped Accept-CH cache=],
+given an |origin| and a [=client hints set=] |hintSet|,
+[=map/set=] [=prerendering browsing context/prerender scoped Accept-CH cache=][|origin|] to |hintSet|.
+
+</div>
+
+<div algorithm="append client hints to request patch">
+  Modify the <a spec=CLIENT-HINTS-INFRASTRUCTURE>append client hints to request</a> algorithm, by prepending the following steps before iterating over the |hintSet|:
+
+  1. Let |browsingContext| be |settingsObject|’s [=global object=]'s [=browsing context=].
```

Comment:
```suggestion
  1. Let |browsingContext| be |settingsObject|’s [=environment settings object/global object=]'s [=Window/browsing context=].
```

Reply:
Hmm, this reveals a potential issue. What if settings object is for a worker, instead of a window?

I think we avoid this issue in the current prerendering spec by delaying all worker loads. So I think we can just bail out if `|settingsObject|'s [=environment settings object/global object=]` is not a `{{Window}}` object.